rollup/base-config.js

```diff
@@ -14,8 +15,9 @@ const plugins = [
 
   replace({
     values: {
-      __DEV__: JSON.stringify(DEV),
-      __DEMO__: JSON.stringify(DEMO),
+      ___DEV___: JSON.stringify(DEV),
```

Comment:
I think that this will not be an issue if we use the Polymer preferred approach.